tests/MailboxTest.php

```diff
@@ -146,6 +146,16 @@ public function testCustomStatusFlag()
         static::assertFalse(isset($status->uidnext), 'uidnext shouldn\'t be set');
     }
 
+    public function testQuota()
+    {
+        static::markTestIncomplete('imap_get_quotaroot isn\'t supported by the current c-client library');
```

Comment:
I've tested the feature and indeed it works against GMail, for example. We should definitively add a real mail provider to out test suite

Reply:
This is not up to you, but I would like to have a working SUT before merging new features

Reply:
I can understand this. Will you upgrade the docker imap server or what is your plan for this?

Reply:
I think I'll create a real GMail account and set its password as a Secret in Travis, so the contributor will not be able to test it inside Docker but has to wait the Travis build

Reply:
In the meantime what you can do is alter the test checking for `getenv('IMAP_QUOTAROOT_SUPPORTED')`: if true do the two `assertArrayHasKey` you wrote, otherwise expect exception for unsupported feature